src/components/HomePage/ContactSection.js

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+
+import {
+    LinkedIn,
+    Facebook,
+    GitHub,
+    Instagram,
+    Twitter,
+    Language,
+    Email,
+} from "@material-ui/icons";
+import { makeStyles } from "@material-ui/core/styles";
+import Theme from "../../AppTheme";
+
+import {
+    Grid,
+    List,
+    ListItem,
+    Button,
+    Typography,
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+    parentContainer: {
+        backgroundColor: Theme.palette.dark.main,
+        color: Theme.palette.dark.contrastText,
+        padding: theme.spacing(4),
+    },
+    colorText: {
+        color: Theme.palette.tertiary.main,
+    },
+    backgroundColor: {
+        color: Theme.palette.dark.main,
+    },
+    primaryColor: {
+        color: Theme.palette.primary.main,
+    },
+}));
+
+
+const ContactSection = () => {
+
+    const classes = useStyles();
+
+    return (
+        <div className={classes.parentContainer}>
+            <Grid container spacing={0}  >
+                <Grid item xs={12} sm={6} md={3}>
+                    <List>
+                        <ListItem>
+                            <Typography variant="h5">Contact us</Typography>
+                        </ListItem>
+                        <ListItem>
+                            <Button
+                                href="mailto:ni@aefeup.pt"
+                                startIcon={<Email htmlColor={Theme.palette.primary.main} />}
```

Comment:
Don't import the theme directly, there can be overrides in place which won't work this way. Instead, you can use the `useTheme()` hook which returns the theme in place here, and you should use that instead.

You can declare it before (similar to the useStyles call), and reuse it whenever you need it in the tree